### ISSUE TITLE
Crash with custom paths

### ISSUE THREAD

**Issue by Cralex** [Kyraminol/3DSync#4]
After revoking my old token and generating a new one for certain reasons, I decided to add some custom paths for good measure. Here's what I added:

NSUI Saves /nsui_forwarders_data
Retroarch Saves /retroarch/cores/savefiles
Retroarch Savestates /retroarch/cores/savestates

Trying to run 3DSync after this resulted in a crash just after it finished warming up. Here's the dump: 
[crash_dump_00000002.zip](https://github.com/Kyraminol/3DSync/files/4666628/crash_dump_00000002.zip)

I guessed that maybe the spaces in the Name field might be causing the problem, so I replaced the spaces with dashes. It still crashed. Note that I first encountered this problem with the debug build you provided, but I also tried the current release and still got a crash. If needed, I could try to get some slow-motion footage of the screen before it crashes.

**Comment by Kyraminol** [comment]
I updated the configurator, try the a new config file. If you can, try to open the new one before testing and see if any path has a trailing slash (like: /retroarch/savestates/ <- this slash shouldn't be there). Previous configurator did add some slashes where they shouldn't be

**Comment by Cralex** [comment]
I created a new config file just a few minutes ago with the same custom paths as last time. So far, it's working fine and none of the paths have a trailing slash. However, my first path had an extra leading slash in the ini file, so the first path appeared as //nsui_forwarders_data . This didn't look right, so I removed the extra slash before running it.
Edit: Yeah, flaky home internet aside, the upload finished without error.
